Formula/crytic-compile.rb

```diff
@@ -6,6 +6,7 @@ class CryticCompile < Formula
   url "https://files.pythonhosted.org/packages/e3/0f/f76d9bc9118d3d330d4d21fdf10ca1c58653aece583c05ae0c9ec4d0dc6a/crytic-compile-0.3.1.tar.gz"
   sha256 "2f0030315b297d1852353b03ace8a484fb0415e07b16ff6172173fbb51313590"
   license "AGPL-3.0-only"
+  revision 1
```

Comment:
Don't bump the revision for test-only changes, it'll force everyone to upgrade, thanks.

Reply:
Good point, I dropped the `revision 1` on the four files now.

Reply:
Thanks!